### ISSUE TITLE
Unregistered element check like the polymer bookmarklet marking them red in the list

### ISSUE THREAD

**Issue by gertcuykens** [beaufortfrancois/polymer-ready-chrome-extension#11]
Example from the polymer bookmarklet:

```js
javascript:(function()%7Bfunction isUnregisteredCustomElement(el)%7Bif(el.constructor%3D%3DHTMLElement)%7Bconsole.error(%27Found unregistered custom element:%27,el)%3Breturn true%3B%7Dreturn false%3B%7Dfunction isCustomEl(el)%7Breturn el.localName.indexOf(%27-%27)!%3D-1%7C%7Cel.getAttribute(%27is%27)%3B%7Dvar allCustomElements%3Ddocument.querySelectorAll(%27html /deep/ *%27)%3BallCustomElements%3DArray.prototype.slice.call(allCustomElements).filter(function(el)%7Breturn isCustomEl(el)%3B%7D)%3Bvar foundSome%3Dfalse%3Bfor(var i%3D0,el%3Bel%3DallCustomElements%5Bi%5D%3B%2B%2Bi)%7Bif(isUnregisteredCustomElement(el))%7BfoundSome%3Dtrue%3B%7D%7Dif(foundSome)%7Balert(%27Oops: found one or more unregistered custom elements in use! Check the console.%27)%3B%7Delse%7Balert(%27Good: All custom elements are registered :)%27)%3B%7D%7D)()%3B
```

**Comment by beaufortfrancois** [comment]
Do you have a URL with unregistered elements I can try this bookmarklet?

**Comment by gertcuykens** [comment]
![image](https://cloud.githubusercontent.com/assets/863731/14248209/8854d304-fa73-11e5-85cf-de953b6c10de.png)

![image](https://cloud.githubusercontent.com/assets/863731/14248240/beafd160-fa73-11e5-9388-245eb1761d0a.png)

code example: http://jsbin.com/kalicomono/edit?html

test bookmarklet opening http://output.jsbin.com/kalicomono and press your bookmarklet

**Comment by beaufortfrancois** [comment]
When I go to http://output.jsbin.com/kalicomono, it says `'Good: All custom elements are registered :)'`
Moreover, I don't think testing for `HTMLElement`  is the right way to check if a custom element is registered or not.

**Comment by beaufortfrancois** [comment]
Here's WIP: https://github.com/beaufortfrancois/polymer-ready-chrome-extension/compare/addUnregisteredElements?expand=1

**Comment by gertcuykens** [comment]
PS You need to open the jsbin output in popup mode apparently for the bookmarklet to work

**Comment by beaufortfrancois** [comment]
Thank you!

Still, I don't think testing for `HTMLElement` is the right way to check if a custom element is registered or not. It fails for some Polymer elements.

**Comment by gertcuykens** [comment]
You are right about that, time to ask the polymer developers for some review advice.
@addyosmani @ebidel @robdodson

**Comment by arthurevans** [comment]
@beaufortfrancois this test shouldn't fail for any _upgraded_ Polymer instance. If the element has been registered, it isn't a vanilla HTMLElement any more. Can you give me an example of where it fails?

**Comment by beaufortfrancois** [comment]
It fails for 99% Polymer Elements at https://www.polymer-project.org/1.0/docs/start/quick-tour.html

~~Elements~~ are the ones not registered in the screenshot below: 
![screenshot 2016-04-07 at 8 38 16 am](https://cloud.githubusercontent.com/assets/634478/14342733/281b2e68-fc9c-11e5-9f7c-1e698017ad5c.png)

**Comment by ebidel** [comment]
This bookmarklet is from https://gist.github.com/ebidel/cea24a0c4fdcda8f8af2. Checking if a viable custom element name ("-" in it) has the `HTMLElement` as its constructor is the way to know if an element had been upgraded.

https://www.polymer-project.org/1.0/docs/start/quick-tour.html doesn't give me any false positives.

**Comment by arthurevans** [comment]
There's something rather odd there, to be sure, because some of those elements (`name-tag`, `picture-frame`) only occur in iframes, and so aren't registered (or used) in the main document.

(I believe was filed before our new web site rolled out, hence the obsolete `core-*` elements... But I can't reproduce these results on either the new or the old site.
